common/src/main/java/grakkit/Instance.java

```diff
@@ -14,7 +14,11 @@ public class Instance {
    public Context context;
 
    /** The engine used for all instance contexts. */
-   public static final Engine engine = Engine.newBuilder().option("engine.WarnInterpreterOnly", "false").build();
+   public static final Engine engine = Engine.newBuilder()
+   .option("engine.WarnInterpreterOnly", "false")
+   .option("inspect", "localhost:9229")
```

Comment:
might be a good idea to make this some kind of config option, iirc the inspect thing halts js until a debugger is attached

Reply:
> iirc the inspect thing halts js until a debugger is attached

Yea, it does. I haven't messed with it too much.